diff/diff.go

```diff
@@ -64,8 +66,63 @@ func Manifests(oldIndex, newIndex map[string]*manifest.MappingResult, options *O
 		doDiff(&report, key, nil, newContent, options)
 	}
 
+	filteredReport := Report{}
+	if len(options.SuppressedOutputLineRegex) > 0 {
+		filteredReport.format = report.format
+		filteredReport.entries = []ReportEntry{}
+
+		var suppressOutputRegexes []*regexp.Regexp
+
+		for _, suppressOutputRegex := range options.SuppressedOutputLineRegex {
+			regexp, err := regexp.Compile(suppressOutputRegex)
+			if err != nil {
+				panic(err)
+			}
+
+			suppressOutputRegexes = append(suppressOutputRegexes, regexp)
+		}
+
+		for _, entry := range report.entries {
+			var diffs []difflib.DiffRecord
+
+			for _, diff := range entry.diffs {
+				matched := false
+
+				for _, suppressOutputRegex := range suppressOutputRegexes {
+					if suppressOutputRegex.MatchString(diff.Payload) {
+						matched = true
+						break
+					}
+				}
+
+				if !matched {
+					diffs = append(diffs, diff)
+				}
+			}
```

Comment:
Perhaps you meant to say / this can be simplified to:

```suggestion
		DIFFS:
			for _, diff := range entry.diffs {
				for _, suppressOutputRegex := range suppressOutputRegexes {
					if suppressOutputRegex.MatchString(diff.Payload) {
						continue DIFFS
					}
				}

				diffs = append(diffs, diff)
			}
```